tests/dummy/lib/template-stuff/index.js

```diff
@@ -11,10 +11,21 @@ module.exports = {
     return true;
   },
 
+  init(parent) {
+    this._super.init && this._super.init.apply(this, arguments);
+
+    let project = parent;
+    while (project.parent) {
+      project = project.parent;
+    }
```

Comment:
Since the second parameter to [`init(parent, project)`](https://ember-cli.com/api/classes/Addon.html#method_init) is deprecated I'm using this `while` loop to traverse the tree up to the root, which must be the project.

Is this how the cool kids do it or is there a more elegant solution? 😂

Reply:
That I'll definitely work, though `this.project` is also an option 😉

Reply:
I couldn't find any explicit docs on `this project` so I assumed it to be private. Gonna change that then :)